pkg/kubelet/cm/cpumanager/policy_static_test.go

```diff
@@ -367,7 +370,7 @@ func TestStaticPolicyAdd(t *testing.T) {
 			pod:         makePod("76000m", "76000m"),
 			expErr:      nil,
 			expCPUAlloc: true,
-			expCSet:     largeTopoSock1CPUSet.Union(cpuset.NewCPUSet(10, 34, 22, 47)),
+			expCSet:     largeTopoSock1CPUSet.UnionAll(sock1CpuSets),
```

Comment:
Please remove the changes to this file.

Reply:
@ConnorDoyle 
I have dropped this change.